components/cards.css

```diff
@@ -0,0 +1,213 @@
+    font-weight: bold;
+}
+
+.prod-content {
+    padding: 8px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: flex-start;
+    gap: 0.8rem;
+}
+
+.prod-info {
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+    width: 100%;
+}
+
+.btn-badge {
+    color: var(--background-color);
+    font-size: smaller;
+    border-radius: 5px;
+    font-weight: 600;
+    padding: 8px;
+    width: max-content;
+    position: absolute;
+    bottom: 4px;
+    left: 0;
+}
+
+.blue {
+    background-color: var(--accent-color);
+}
+
+.green {
+    background-color: var(--badge-color-green);
+}
+
+.ratings-con {
+    margin-left: auto;
+    color: var(--hover-color-links);
+    font-size: 0.5rem;
+}
+
+.prod-links .prod-btn i {
+    padding: 0;
+    font-size: 0.9rem;
+}
+
+.prod-btn {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+
+.prod-btn .prod-like .fal {
+    font-size: 2rem;
+}
+
+.prod-btn .btn {
+    padding-left: 3rem;
+    padding-right: 3rem;
+}
+
+.prod-price {
+    color: var(--cta-color);
+}
+
+.size-options {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    gap: 5px;
+    margin: 5px;
+}
+
+.size-options button {
+    padding: 5px 8px;
+    background-color: var(--font-color-grey);
+    border-radius: 50%;
+    font-weight: 600;
+}
+
+.size-options button:active {
+    border: 1px solid var(--size-active);
+}
+
+.btn-badge-dismiss {
+    color: var(--background-color);
+    font-size: smaller;
+    border-radius: 5px;
+    font-weight: 600;
+    padding: 5px;
+    width: max-content;
+    position: absolute;
+    top: 4px;
+    right: 0;
+}
+
+.card-overlay {
+    position: relative;
+    pointer-events: none;
+}
+
+.overlay-bg {
+    opacity: 0.6;
+}
+
+.text-overlay {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 1rem;
+    background-color: var(--badge-background-overlay);
+}
+
+.text-overlay div {
+    padding: 8px;
+    border-radius: 5px;
+    font-size: #000;
+    background-color: var(--background-color);
+}
+
+.text-card {
+    width: 35rem;
+    padding: 1rem;
+    box-shadow: var(--box-shadow-card);
+}
+
+.text-card-title h1 {
+    font-weight: 600;
+    color: var(--accent-color);
+}
+
+.text-card-content {
+    text-align: justify;
+    font-size: 1.1rem;
+    line-height: 1.3;
+    letter-spacing: 1px;
+}
+
+/* horizontal card */
+
+.card-horizontal {
+    background-color: var(--badge-background);
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    height: 12.5rem;
+    padding: 0.8rem;
+    border-radius: 8px;
+}
+
+.card-horizontal .product {
+    flex-basis: 20%;
+}
+
+.card-horizontal .prod-links {
+    flex-grow: 1;
+}
+
+.card-horizontal .prod-links .prod-content .prod-info {
+    justify-content: flex-start;
```

Comment:
You can use this as a utility class named as .native-justify-flex-start with these properties and pass it into the HTML wherever needed!!